engine/core/action.lua

```diff
@@ -4,27 +4,27 @@
 --- !doc protected-members
 --- @class Action : Object
 --- @field owner Actor The actor taking the action.
---- @field protected targets Target[] (static) A list of targets to apply the action to.
---- @field protected targetObjects Object[] The objects that correspond to the targets.
+--- @field protected targetObjects Target[] (static) A list of targets to apply the action to.
+--- @field protected targets any[] The objects that correspond to the targets.
```

Comment:
This **does** break the API, since you do `Action.targets = {}` to define them. Basically every `Action` would break.